mlflow/pyfunc/__init__.py

```diff
@@ -716,6 +724,18 @@ def load_model(
     return PyFuncModel(model_meta=model_meta, model_impl=model_impl)
 
 
+def print_model_dependencies(model_uri):
+    """
+    Given a model URL, print model dependencies provided by model’s requirements.txt file.
+    """
+    local_path = _download_artifact_from_uri(artifact_uri=model_uri, output_path=None)
+    req_file_path = os.path.join(local_path, _REQUIREMENTS_FILE_NAME)
```

Comment:
What happens if the model doesn't have a `requirements.txt` file? Also, what happens if the user wants to get conda dependencies instead of just pip dependencies? We should support this.